third_party/terraform/resources/resource_dataflow_job.go

```diff
@@ -245,6 +298,57 @@ func resourceDataflowJobRead(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+// Stream update method. Batch job changes should have been set to ForceNew via custom diff
+func resourceDataflowJobUpdateByReplacement(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	region, err := getRegion(d, config)
+	if err != nil {
+		return err
+	}
+
+	zone, err := getZone(d, config)
+	if err != nil {
+		return err
+	}
+
+	params := expandStringMap(d, "parameters")
+	labels := expandStringMap(d, "labels")
+
+	env := dataflow.RuntimeEnvironment{
```

Comment:
can this be made into a function that can be called from both create and update?